src/components/app/TabBar.css

```diff
@@ -44,18 +44,20 @@
 }
 
 /* Place the top tab bar in a separate element to avoid tapered corners. */
-.tabBarTab::before {
-  position: absolute;
-  top: 0;
-  right: -1px;
-  left: -1px;
-  height: 2px;
-  background-color: transparent;
-  content: '';
-  transition:
-    background-color 200ms,
-    border-color 200ms;
-  transition-timing-function: var(--animation-timing);
+@media (forced-colors: none) {
```

Comment:
shouldn't we keep that one outside of a `@media`, for these browsers that don't support `force-colors`?

Reply:
Sure, I'll try to handle it